tests/test_core.py

```diff
@@ -71,9 +69,9 @@ def test_get_blocks_status_containing_tx(monkeypatch):
         {'id': 1}, {'id': 2}
     ]
     monkeypatch.setattr(
-        Bigchain, 'search_block_election_on_index', lambda x, y, z: blocks)
+        Bigchain, 'search_block_election_on_index', lambda x, y: blocks)
     monkeypatch.setattr(
-        Bigchain, 'block_election_status', lambda x, y: Bigchain.BLOCK_VALID)
+        Bigchain, 'block_election_status', lambda x, y, z: Bigchain.BLOCK_VALID)
```

Comment:
@sohkai see https://github.com/sbellem/bigchaindb/blob/a14f572a0f127f25ba015b428f74b46e108a50f1/bigchaindb/core.py#L686

`x, y, z` refer to `self, block_id, voters`